### ISSUE TITLE
dfw_draw_centered has no effect

### ISSUE THREAD

**Issue by brianfleming** [aziz/DistractionFreeWindow#23]
I'm trying to get the centered text setting to have an effect. My settings are:

```json
    "dfw_draw_centered": true,
    "dfw_wrap_width": 90,
```

When i toggle into DistractionFreeWindow mode there is no difference whether `dfw_draw_centered` is `true` or `false`.

### Environment

```text
DistractionFreeWindow:
    version:        0.6.5
Sublime Text:
    channel:        dev
    version:        Build 3125
    platform:       macOS Sierra
    architecture:   x64
```
what am I doing wrong?

**Comment by jrappen** [comment]
@brianfleming I'll review this later today. Just to confirm, you are using `v0.6.5` of DistractionFreeWindow?

**Comment by brianfleming** [comment]
@jrappen yes I am. I had just installed via package control:

![screen shot 2017-02-16 at 8 32 00 am](https://cloud.githubusercontent.com/assets/418001/23023210/694a1ae2-f422-11e6-855b-97996806b88b.png)

**Comment by jrappen** [comment]
@brianfleming Did you set that in `ST3/Data/Packages/User/Preferences.sublime-settings`? It must be in `ST3/Data/Packages/User/distraction_free_window.sublime-settings`.

**Comment by brianfleming** [comment]
@jrappen I did. I used the menu path:
![path](https://cloud.githubusercontent.com/assets/418001/23044635/1147cf18-f46f-11e6-9518-b2722f11797f.png)

and here i am hovering over the tab so you can see the file path:

![settigns](https://cloud.githubusercontent.com/assets/418001/23044644/1b1b7788-f46f-11e6-9c32-dbd3fd4c2d85.png)

**Comment by jrappen** [comment]
@brianfleming Do the `"dfw_hide_*"` options work fine when toggling?

**Comment by brianfleming** [comment]
@jrappen looks like some of them do, here it is when i toggle the feature:

![screen shot 2017-02-16 at 6 01 53 pm](https://cloud.githubusercontent.com/assets/418001/23045270/04d07502-f472-11e6-8d6e-e0878a5c5ed9.png)


just scanning it seems like these are the ones that are not working:
```
"dfw_hide_rulers": true,
"dfw_hide_indent_guides": true,
"dfw_hide_line_numbers": true,
"dfw_hide_fold_buttons": false,
"dfw_hide_gutter": false,
```

**Comment by jrappen** [comment]
@brianfleming Seems like none of them work.

**Comment by brianfleming** [comment]
i thought these are behaving as desired:

```
    "dfw_hide_tabs": true,
    "dfw_hide_status_bar": true,
    "dfw_hide_minimap": true,
    "dfw_hide_side_bar": true,
    "dfw_hide_menu": true,
```

**Comment by jrappen** [comment]
@brianfleming Sorry. Yes, was ignoring those few. I'll look into it tomorrow again. Seems like the problem is [aziz/DistractionFreeWindow`/distraction_free_window.py`#L128-L140](https://github.com/aziz/DistractionFreeWindow/blob/bcecb52745be81bdef247aced61afc62ad47f6be/distraction_free_window.py#L128-L140)

**Comment by brianfleming** [comment]
No worries @jrappen thanks for the prompt replies. I am really enjoying the plugin even without those settings.

**Comment by jrappen** [comment]
@brianfleming I think I found it, I assume it works with ST2? Could you try that?

**Comment by brianfleming** [comment]
@jrappen yes it does work.
I just downloaded ST2 and installed and it worked with what I think is the latest ST2 version:

```
version: 2.0.2
build: 2221
```

<details>
<summary>screenshot Sublime Text 2: dfw_settings</summary>

<img width="1442" alt="screen shot 2017-02-19 at 6 01 22 pm" src="https://cloud.githubusercontent.com/assets/418001/23107583/89614d84-f6cd-11e6-8236-adf72429d660.png">


</details>

**Comment by jrappen** [comment]
@brianfleming Should be resolved in aziz/DistractionFreeWindow@a5dc567, which is `v0.6.6`. Should problems persist, please re-open.
